integration-tests/integration/reportPages/enter-incident-details.spec.js

```diff
@@ -243,4 +244,12 @@ context('Submitting details page form', () => {
       .name()
       .should('not.exist')
   })
+  it('Displays the current prison', () => {
+    cy.login(bookingId)
+    fillFormAndSave()
+    cy.go('back')
```

Comment:
would avoid adding a specific test to this (ideally keep build times down) - could add this to `fillFormAndSave` when we check the offender names is set

Reply:
done